### PR TITLE
CB: Fix Sort

### PR DESCRIPTION
**Pull request by drostron**
addresses: #1654

### CODE REVIEW

File: couchbase/src/test/scala/quasar/physical/couchbase/BasicQueryEnablementSpec.scala [hunk 1]
```diff
@@ -235,20 +235,6 @@ class BasicQueryEnablementSpec
 
       n1ql must_= """select value v from (select value _2 from (select value _0 from (select value ifmissing(v.`value`, v) from `foo` v) as _1 unnest _1 as _0) as _2 where _2.["bar"] >= 1 and _2.["bar"] <= 10) as v"""
     }
-
-    "convert a basic order by" in {
```
Comment (alissapajer) [inline couchbase/src/test/scala/quasar/physical/couchbase/BasicQueryEnablementSpec.scala]: Is this test overlapped somewhere else? Or is there another reason it's been deleted?

File: it/src/main/resources/tests/sortProjectLimit.test [hunk 2]
```diff
@@ -3,8 +3,7 @@
 
     "backends": {
         "postgresql": "pending",
-        "marklogic":  "skip",
-        "couchbase":  "skip"
+        "marklogic":  "skip"
```
Comment (alissapajer) [inline it/src/main/resources/tests/sortProjectLimit.test]: Is this test (and "largestCities.test" above) meant to still be skipped?

Comment (alissapajer) [inline it/src/main/resources/tests/sortProjectLimit.test]: And the follow-up would be: are there any tests you can enable to show this fix works?

Comment (drostron) [inline it/src/main/resources/tests/sortProjectLimit.test]: Meant to remove skip for couchbase and meant to show that this fix works. 😄

File: couchbase/src/test/scala/quasar/physical/couchbase/BasicQueryEnablementSpec.scala (hunk 1 above)
Comment (drostron) [inline couchbase/src/test/scala/quasar/physical/couchbase/BasicQueryEnablementSpec.scala]: Removing for the same reason as in the previous PR and overlapped by the freshly enabled query integration tests.

**Comment by alissapajer** [comment]
@drostron also, this needs a changelog.

**Comment by drostron** [comment]
@alissapajer Will add with `[skip ci]` after underway build completes, as to not collapse its status here.

**Comment by drostron** [comment]
Added and ready. Status was collapsed anyway — [passing build here](https://travis-ci.org/quasar-analytics/quasar/builds/172069885).

**Comment by alissapajer** [comment]
@drostron I still see the two integration test marked as skipped.

**Comment by alissapajer** [comment]
^^ re the integration tests, I misread the diff :) all is good, will merge!

**Pull request merged**